src/aaudio/AudioStreamAAudio.cpp

```diff
@@ -981,4 +983,44 @@ Result AudioStreamAAudio::setOffloadEndOfStream() {
     return static_cast<Result>(mLibLoader->stream_setOffloadEndOfStream(stream));
 }
 
+void AudioStreamAAudio::updateDeviceIds() {
```

Comment:
Please document any guarantees about the final state of mDeviceIds?

Reply:
Added a warning when mDeviceIds is empty